CSV_REFLECT/src/main/java/org/example/generator/manager/formats/csv/CsvGenerator.java

```diff
@@ -1,122 +0,0 @@
-package org.example.generator.manager.formats.csv;
-
-import org.example.annotations.*;
-import org.example.generator.data.ClassFields;
-import org.example.generator.data.GeneratorData;
-import org.example.generator.manager.interfaces.Generator;
-
-import java.io.BufferedWriter;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.lang.reflect.Field;
-import java.time.LocalDate;
-import java.time.format.DateTimeFormatter;
-import java.util.List;
-
-public class CsvGenerator implements Generator {
-    private final String SPLITTER = "; ";
-
-
-    @Override
-    public void generate(String targetLoc, ClassFields classFields, GeneratorData generatorData) throws IOException {
-        GeneratorData csvData = generatorData;
-        ClassFields csvClassFields = classFields;
-
-        FileWriter scan = new FileWriter(targetLoc + ".csv");
-        BufferedWriter writer = new BufferedWriter(scan);
-
-        System.out.println();
-        getColTitles(csvData, csvClassFields, writer);
-        getRowValues(csvData, csvClassFields, writer);
-
-        writer.close();
-        scan.close();
-
-    }
-
-
-    public void getColTitles(GeneratorData csvData, ClassFields classFields, BufferedWriter writer) throws IOException {
-        StringBuilder colTitle = new StringBuilder();
-
-
-        classFields.getListOfClassFields().forEach(field -> {
-            if (!csvData.getListOfObjects().get(0).getClass().isAnnotationPresent(DontGenerate.class))
-                if (!field.isAnnotationPresent(IgnoreField.class))
-                    if (csvData.getListOfObjects().get(0).getClass().isAnnotationPresent(IgnoreInnerLists.class)) {
-                        if (!field.getType().equals(List.class))
-                            colTitle.append(field.getName()).append(SPLITTER);
-                    } else {
-                        colTitle.append(field.getName()).append(SPLITTER);
-                    }
-
-
-        });
-
-
-        colTitle.setLength(colTitle.length() - 2);
-        colTitle.append("\r");
-
-        writer.write(colTitle.toString());
-        System.out.println(colTitle);
-
-
-    }
-
-    public void getRowValues(GeneratorData csvData, ClassFields classFields, BufferedWriter writer) {
-
-
-        if (!csvData.getListOfObjects().get(0).getClass().isAnnotationPresent(DontGenerate.class))
-            csvData.getListOfObjects().forEach(obj -> {
-                StringBuilder sb = new StringBuilder();
-
-                for (Field field : classFields.getListOfClassFields()) {
-                    Object fieldValue;
-
-                    try {
-                        field.setAccessible(true);
-                        fieldValue = field.get(obj);
-
-                        if (csvData.getListOfObjects().get(0).getClass().isAnnotationPresent(NullsEquals.class))
```

Comment:
Przez tą masę ifów jest to mało czytelne. Spróbuj zrobić metodę getFieldValue, która będzie zwracała to co ma zostać appendowane.